pkg/dashboard/apiserver/auth/gcp/fx.go

```diff
@@ -0,0 +1,27 @@
+// Copyright Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package gcp
+
+import (
+	"github.com/go-logr/logr"
+
+	config "github.com/chaos-mesh/chaos-mesh/pkg/config/dashboard"
+)
+
+func Bootstrap(conf *config.ChaosDashboardConfig, logger logr.Logger) *Service {
+	service, _ := NewService(conf, logger.WithName("gcp auth api"))
+	return service
+}
```

Comment:
please do not ignore that error returned by `NewService`. After that, `Bootstrap` do not do extra things other than `NewService`, please use `NewService` with `fx` directly.

Reply:
Thanks, I'll apply.